src/shared/infrastructure/profile-metrics/profile-metrics.json

```diff
@@ -43,28 +43,28 @@
   },
   "memory:alloc_objects:count:space:bytes": {
     "id": "memory:alloc_objects:count:space:bytes",
-    "description": "Total objects allocated",
+    "description": "Number of objects allocated",
     "type": "alloc_objects",
     "group": "memory",
     "unit": "short"
   },
   "memory:alloc_space:bytes:space:bytes": {
     "id": "memory:alloc_space:bytes:space:bytes",
-    "description": "Total bytes allocated in the heap",
+    "description": "Size of memory allocated in the heap",
     "type": "alloc_space",
     "group": "memory",
     "unit": "bytes"
   },
   "memory:inuse_objects:count:space:bytes": {
     "id": "memory:inuse_objects:count:space:bytes",
-    "description": "Count of objects currently in use",
+    "description": "Number of objects currently in use",
```

Comment:
Do number and count imply different actions or are they good synonyms? I know in some context a count implies a different action than number. That might not matter here.

Reply:
Good question, I don't know 🤷🏾‍♂️

Reply:
Means the same to me